cloud-compose/templates/datadog.docker.sh

```diff
@@ -1,5 +1,9 @@
-# datadog.docker.sh 
+# datadog.docker.sh
+{%- if secrets.DATADOG_API_KEY is defined %}
```

Comment:
it should be like this:
{%- if secrets is defined and secrets.DATADOG_API_KEY is defined %}